standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/messaging/event/filters/ReplEventFilter.java

```diff
@@ -30,12 +31,22 @@ public ReplEventFilter(final ReplScope replScope) {
     this.replScope = replScope;
   }
 
+  // Return false if all the tables are included, as bootstrap during rename is done only in case filter set for tables.
+  boolean isAlterDuringTableLevelReplication(final NotificationEvent event) {
+    if (replScope.includeAllTables() || !replScope.getDbNamePattern().matcher(event.getDbName()).matches()) {
```

Comment:
if db name can be exposed ..this i don't think has any issue ..

Reply:
Well. my point is not about exposing dbNamePattern. It's more about functionality related to ReplScope. So, the db matching is a functionality which can be kept isolated within ReplScope. Why need to expose this functionality outside? Also, having it inside ReplScope would make others to use it in future instead of duplicating it in the caller.
Anyways, it's your call.

Reply:
shall use dbIncludedInReplScope